File-Formats/DocIO/word-to-image.md

```diff
@@ -332,6 +332,319 @@ End Using
+         {
+             //Reset the stream position.
+             stream.Position = 0;
+             //Save the stream as file.
+             using (FileStream fileStreamOutput = File.Create("WordToImage_" + i + ".jpeg"))
+		     {
+		         stream.CopyTo(fileStreamOutput);
+		     }
+		     i++;
+		 }
+      }
+   }
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+### Fallback fonts based on script type
+
+The following code example demonstrates how a user can add fallback fonts based on the script types, which DocIO considers internally when converting a Word document to Image.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+//Opens the file as stream
+using (FileStream inputStream = new FileStream("Template.docx", FileMode.Open, FileAccess.Read))
+{
+   //Loads an existing Word document file stream
+   using (WordDocument wordDocument = new WordDocument(inputStream, Syncfusion.DocIO.FormatType.Docx))
+   {
+      //Adds fallback font for "Arabic" script type
+      wordDocument.FontSettings.FallbackFonts.Add(ScriptType.Arabic, "Arial, Times New Roman, Courier New, Segoe UI, Tahoma, Traditional Arabic");
+      //Adds fallback font for "Hebrew" script type
+      wordDocument.FontSettings.FallbackFonts.Add(ScriptType.Hebrew, "Arial, Times New Roman, David, Courier New, Microsoft Sans Serif");
+      //Adds fallback font for "Hindi" script type
+      wordDocument.FontSettings.FallbackFonts.Add(ScriptType.Hindi, "Mangal, Nirmala UI, Utsaah");
+      //Adds fallback font for "Chinese" script type
+      wordDocument.FontSettings.FallbackFonts.Add(ScriptType.Chinese, "DengXian, MingLiU, MS Gothic, SimSun");
+      //Adds fallback font for "Japanese" script type
+      wordDocument.FontSettings.FallbackFonts.Add(ScriptType.Japanese, "Yu Mincho, MS Mincho, MS Gothic");
+      //Adds fallback font for "Thai" script type
+      wordDocument.FontSettings.FallbackFonts.Add(ScriptType.Thai, "Tahoma, Microsoft Sans Serif");
+      //Adds fallback font for "Korean" script type
+      wordDocument.FontSettings.FallbackFonts.Add(ScriptType.Korean, "Malgun Gothic, Batang, Dotum, Gulim");
+      //Instantiation of DocIORenderer for Word to PDF conversion.
+      using (DocIORenderer render = new DocIORenderer())
+      {
+		 //Convert the entire Word document to images.
+         Stream[] imageStreams = wordDocument.RenderAsImages(); 
+		 int i = 0;
+         foreach (Stream stream in imageStreams)
+		 {
+             //Reset the stream position.
+		     stream.Position = 0;
+             //Save the stream as file.
+		     using (FileStream fileStreamOutput = File.Create("WordToImage_" + i + ".jpeg"))
+             {
+		         stream.CopyTo(fileStreamOutput);
+             }
+		     i++;
+         }
+      }
+   }
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+### Fallback fonts for range of Unicode text
+
+Users can set fallback fonts for their custom Unicode range of text to be used in Word to Image conversion.
```

Comment:
@LokeshBaskar - `custom Unicode range` is not a valid name. Because, our fallback font feature only works with our known Unicode range.

Reply:
Modified the sentences as known Unicode range of text